### PR TITLE
docs: fix Sphinx build

### PR DESCRIPTION
**Pull request by kentbull**
Upgrade Sphinx to 8.1.3 and myst-parser to 4.1.3.

1. doing `make clean html` builds the docs.
2. Open the docs on OSx with `open _build/html/index.html`
3. Profit

 There are a bunch of warnings yet they don't stop the docs from building.

### CODE REVIEW

**Comment by kentbull** [comment]
took about 15 mins of work. What else, if anything, is missing from this? the docs build into the static assets `_build` dir and can be published. It seems complete to me.

**Review by m00sey — COMMENTED** [review]
seems to be missing the vdr package

**Comment by m00sey** [comment]
Appreciate it only took 15 minutes.

The local rendering for me does not have any detail, screenshot attached.

I assume this is just a templating issue, but it is what I referred to on the call.

Screenshot 2024-10-15 at 4 31 07 PM

**Comment by kentbull** [comment]
I'm also seeing the VDR API:

image

And the details of the VDR API:

image

This is when I do an `open ./_build/html/index.html` from the OSx command line after a `make clean html`

**Comment by m00sey** [comment]
LGTM - however I dislike the theme lol

File: docs/conf.py
Comment (m00sey) [inline docs/conf.py:1]: Think the copyright needs to be 2022 - 2024 inclusive of previous years, not exclude them?

File: docs/index.rst [hunk 1]
```diff
@@ -35,11 +35,13 @@ API Reference
     keri_app
     keri_core
     keri_db
+    keri_demo
```
Comment (m00sey) [inline docs/index.rst:38]: I think we should remove this

Comment (kentbull) [reply docs/index.rst]: This is removed.

File: docs/conf.py
Comment (kentbull) [reply docs/conf.py:1]: Should it be copyright or should it be creative commons or Apache 2 or something?

Comment (kentbull) [reply docs/conf.py:1]: for now I changed it to 2022 - 2024

**Pull request merged**